doc/user/content/releases/v0.64.md

```diff
@@ -1,7 +1,8 @@
 ---
 title: "Materialize v0.64"
 date: 2023-08-09
-released: false
+released: true
+patch: 0
```

Comment:
```suggestion
```

Reply:
This isn't necessary until there have been minor releases. I'm...mostly unsure about how CI would take `patch: 0`, so we can either remove it or find out.